bin/kibana-plugin

```diff
@@ -21,4 +21,4 @@ if [ ! -x "$NODE" ]; then
   exit 1
 fi
 
-exec "${NODE}" $NODE_OPTIONS "${DIR}/src/cli_plugin" ${@}
+exec "${NODE}" $NODE_OPTIONS --no-warnings "${DIR}/src/cli_plugin" ${@}
```

Comment:
The cli plugin command has it's own logger with it's own opinions about when messages should and shouldn't be logged (controlled with `--silent` and `--quiet`) https://github.com/elastic/kibana/blob/master/src/cli_plugin/lib/logger.js

We should probably wire this up in that cli